### ISSUE TITLE
SOMD example config gives incorrect units for parameters

### ISSUE THREAD

**Issue by lohedges** [michellab/Sire#229]
SOMD can be used to generate an example configuration file with default parameters, i.e.

```bash
$HOME/sire.app/bin/somd --help-config
```

However, for parameters that have units, e.g. times, temperatures, etc., the unit names are incorrect and SOMD will not run if these are used. This is because SOMD uses `Sire.Units` for unit handling and the unit names must match the _exact_ names from this package. For example, the configuration output gives the following defaults:

```bash
temperature = 25 C
timestep = 0.002 ps
```

In order for these to work, the names must be changed to:

```
temperature = 25 celsius
timestep = 0.002 picosecond
```

Also note that the variables are not assigned a default unit if no unit is present, i.e. when the value is just a float or integer. This means that SOMD will fail if the user configuration file doesn't use the _exact_ unit name. (No abbreviations, alternative spellings, etc, are allowed.)

This could be fixed by having some kind of regex handling and unit checking within the `resolveParameters` decorator from [here](https://github.com/michellab/Sire/blob/devel/wrapper/Tools/__init__.py) (the most flexible option), or by updating the ouput of `--help-config` to display the correct unit names (the simplest short-term fix).

**Comment by lohedges** [comment]
It would be good to improve the SOMD configuration file parsing for the 2019.2.0 release. A basic improvement would be not silently ignoring invalid keywords and making sure that the defaults shown in the help output have the correct naming for the units.

**Comment by ppxasjsm** [comment]
Yep that makes sense! I'll try and work on it, when the simulations are running properly.
